matchzoo/datasets/cqa_ql_16/load_data.py

```diff
@@ -0,0 +1,208 @@
+"""CQA-QL-16 data loader."""
+
+import typing
+from pathlib import Path
+
+import keras
+import pandas as pd
+from xml.etree.ElementTree import parse
+
+import matchzoo
+
+
+_train_dev_url = "http://alt.qcri.org/semeval2016/task3/data/uploads/" \
+                 "semeval2016-task3-cqa-ql-traindev-v3.2.zip"
+_test_url = "http://alt.qcri.org/semeval2016/task3/data/uploads/" \
+            "semeval2016_task3_test.zip"
+
+
+def load_data(
+    stage: str = 'train',
+    task: str = 'classification',
+    target_label: str = 'PerfectMatch',
+    return_classes: bool = False,
+    match_type: str = 'question',
+    method: str = 'both',
+) -> typing.Union[matchzoo.DataPack, tuple]:
+    """
+    Load CQA-QL-16 data.
+
+    :param stage: One of `train`, `dev`, and `test`.
+        (default: `train`)
+    :param task: Could be one of `ranking`, `classification` or a
+        :class:`matchzoo.engine.BaseTask` instance. (default: `ranking`)
+    :param target_label: If `ranking`, choose one of classification
+        label as the positive label. (default: `Good`)
+    :param return_classes: `True` to return classes for classification
+        task, `False` otherwise.
+    :param match_type: Matching text types. One of `question`,
+        `answer`, and `external_answer`. (default: `question`)
+    :param method: Train data use method. One of `part1`, `part2`,
```

Comment:
```suggestion
    :param mode: Train data use method. One of `part1`, `part2`,
```
We usually use `mode` for this kind of thing.